src/app/public/index.ts

```diff
@@ -1 +1 @@
-export * from './src/library.module';
+export * from './modules';
```

Comment:
Probably would be a good idea to list out every type you wish to export, explicitly.

Reply:
Done.